packages/generator/src/generator.js

```diff
@@ -24,15 +25,20 @@ export default class Generator {
 
   async generate ({ build = true, init = true } = {}) {
     consola.debug('Initializing generator...')
-
     await this.initiate({ build, init })
 
-    consola.debug('Preparing routes for generate...')
+    // Payloads for full static
+    if (this.isFullStatic) {
+      consola.info('Full static mode activated')
+      const exportTime = String(Date.now())
+      this.payloadDir = path.join(this.distNuxtPath, 'payloads', exportTime)
```

Comment:
Is there a particular reason why we don't nest it inside `dist/assets/` instead of "just" `/dist/`? ☺️

Reply:
I prefer to separate them from the `assets` directory actually for better logic separation, also it open the possibility to upload only the `payloads` when running `nuxt export` again